README.md

```diff
@@ -446,6 +446,20 @@ var showdown    = require('showdown'),
     converter = new showdown.Converter({ extensions: ['myExtension'] });
 ```
 
+## Building
+
+Building your clone of the repository is easy.
+> Prerequesites: [npm](https://www.npmjs.com/package/npm) and [npx](https://www.npmjs.com/package/npx) must be installed.
+
+1. run `npm install`: installs all the needed third-party packages into the [node_modules](/node_modules)  folder, based on the contents of [package-lock.json](/package-lock.json) and [package.json](/package.json).
+1. run `npx grunt build`(see [Gruntfile.js](/Gruntfile.js)). This:
```

Comment:
```suggestion
1. run `npx grunt build` (see [`Gruntfile.js`](/Gruntfile.js)). This command:
```

Reply:
Sure you want to put backquotes inside a link caption? Sounds a bit strange to me.
And if we do that, I think we want to do the same for the link to the dist folder, just below?

Reply:
It's just a way to highlight important words, filenames, and commands.
To keep things consistent, it's better to highlight the `dist` folder as well. Thanks for spotting this.